src/gluonts/time_feature/lag.py

```diff
@@ -85,9 +85,14 @@ def _make_lags_for_month(multiple, num_cycles=3):
     # multiple, granularity = get_granularity(freq_str)
     offset = to_offset(freq_str)
 
-    if offset.name == "M":
+    if offset.name == "A-DEC":  # Y
+        lags = []
+    elif offset.name == "Q-DEC":  # Q
+        assert offset.n == 1
```

Comment:
Can we add a message here?

Reply:
You mean a message saying that we don't use features for years?
I think that is just the correct behavior. There are no useful features for years I think

Reply:
Sorry, I meant to the assert statement.